lib/pyfrc/physics/drivetrains.py

```diff
@@ -1,3 +1,4 @@
+import math
```

Comment:
This import should go _under_ the docstring — having statements before a docstring makes the docstring… not a docstring.